tests/projects/test_databricks.py

```diff
@@ -470,9 +470,10 @@ def confirm_request_params(*args, **kwargs):
 
 @mock.patch("mlflow.utils.databricks_utils.get_databricks_host_creds")
 def test_run_databricks_failed(_):
-    with mock.patch("mlflow.utils.rest_utils.http_request") as m:
-        text = '{"error_code": "RESOURCE_DOES_NOT_EXIST", "message": "Node type not supported"}'
-        m.return_value = mock.Mock(text=text, status_code=400)
+    text = '{"error_code": "RESOURCE_DOES_NOT_EXIST", "message": "Node type not supported"}'
+    with mock.patch(
+        "mlflow.utils.rest_utils.http_request", return_value=mock.Mock(text=text, status_code=400)
+    ):
```

Comment:
/ai --suggest remove the variable `text` and directly pass the string to the `text` argument in the following code:

Reply:
Can this ai apply formatting with black as well?